Tools/AP_Periph/Parameters.cpp

```diff
@@ -302,6 +306,17 @@ const AP_Param::Info AP_Periph_FW::var_info[] = {
     // @User: Advanced
     // @RebootRequired: True
     GSCALAR(esc_pwm_type, "ESC_PWM_TYPE",     0),
+
+#if HAL_WITH_ESC_TELEM
```

Comment:
```suggestion
#if HAL_WITH_ESC_TELEM && !HAL_GCS_ENABLED
```

Don't need this if we have a serial manager

Reply:
fixed